src/wp-content/themes/twentytwentyone/functions.php

```diff
@@ -424,7 +424,7 @@ function twenty_twenty_one_scripts() {
 		array( 'in_footer' => true )
 	);
 
-	// Register the IE11 polyfill loader.
+	// Register the IE11 polyfill loader. This is no longer used but it is retained if any child theme or plugin uses them.
```

Comment:
I removed `twenty-twenty-one-ie11-polyfills` from being a dependency for the other scripts.